src/main/java/sirius/db/mongo/Finder.java

```diff
@@ -412,7 +415,49 @@ public Value aggregateIn(@Nonnull String collection, @Nonnull Mapping field, @No
                 w.submitMicroTiming(KEY_MONGO,
                                     "AGGREGATE - " + collection + "." + field + " (" + operator + "): " + filterObject);
             }
-            traceIfRequired(collection, w);
+            traceIfRequired("aggregate-" + collection, w);
+        }
+    }
+
+    /**
+     * Executes the given list of facets using the filters specified for this query.
+     *
+     * @param descriptor the entity descriptor to query
+     * @param facets     the facets to execute
+     */
+    public void executeFacets(@Nonnull EntityDescriptor descriptor, List<MongoFacet> facets) {
+        if (facets.isEmpty()) {
```

Comment:
Could be expanded to `facets == null || facets.isEmpty()` which means the if check in `MongoQuery::executeFacets` could be removed